File: Ejercicio/GO/JSON/MostrarJSON.go
package main

import (
	"encoding/json"
	"fmt"
)

func main() {
	persona := map[string]interface{}{
		"nombre": "Valery",
		"edad":   22,
		"ciudad": "Bogotá",
	}

	jsonData, _ := json.Marshal(persona)
	fmt.Println(string(jsonData))
}

File: Ejercicio/Java/JSON/MostrarJSON.java
import org.json.JSONObject;

public class MostrarJSON {
    public static void main(String[] args) {
        JSONObject persona = new JSONObject();
        persona.put("nombre", "Valery");
        persona.put("edad", 22);
        persona.put("ciudad", "Bogotá");

        System.out.println(persona);
        System.out.println("Nombre: " + persona.getString("nombre"));
    }
}
